### DISCUSSION TITLE
How to view / verify the query-params in Visor?

### DISCUSSION THREAD

**Discussion by suntong** [deta/cloud-docs#555] [Help] [answered]
I passed url-query-params like `/user?id=123&name=abc` to my `app.get("/user", ...`, but when I view my requests in Visor, the Request Payload is always empty:

![image](https://user-images.githubusercontent.com/422244/198123278-b8989b1b-a688-4182-97f6-0fa875d20089.png)

How to view / verify the query-params I passed to my node app in Visor?

**Comment by abdelhai** [comment]
get requests don't have a payload. the params are in the url itself.
payloads are usually extra data like json you send with post/put requests

for reference https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/GET

**Reply by suntong** [reply]
I want to view / verify the query-params in Visor.

Since _"get requests don't have a payload"_, and there is nowhere in Visor showing them currently, it might be a good idea showing the query-params in payload.

**Comment by Jorghe** [comment, marked as answer]
Hello @suntong 
I think the query-params and the _Request Payload_ got confused, as @abdelhai suggested earlier, the request points to the Post method which is usually a .json file.
If you really wanted to see those parameters you can print them and see the in the Visor:
```python
@app.get("/")
def render_home(id: str, name: str  ):
  print(f"{id} - {name}")
  return <p>Ok</p>
```
And then watch the Logs in Deta Visor.
